src/main/java/Main.java

```diff
@@ -4,34 +4,57 @@
 
 public class Main {
     public static void main(String[] args) {
-        System.out.println("----------------------------------객 체 생 성-----------------------------------------");
-        Scv scv1 = new Scv("SCV @1",100, 200);
-        Scv scv2 = new Scv("SCV @2",33, 111);
-        BattleCruiser bc = new BattleCruiser("Battlecruiser @1",50, 100);
-        Marine marine = new Marine("Marine @1",123, 123);
+
+        System.out.println("----------------------------------객 체 생 성---------------------------------------------");
+        Scv scv1 = new Scv("SCV @1", "SCV", 60, 60, 5, "ground",100, 200);
```

Comment:
```suggestion
        Scv scv1 = new Scv();
```

SCV 를 생성할때 생성자로 값을 넘겨주는것 좋습니다.
하지만, SCV 라는 클래스가 구체화 된 상태에서, 
해당 클래스에 기본값이 들어가있고, 우린 SCV만 만들면 알아서 값이 채워지는게 더 좋겠죠?
일종의 템플릿이라 이해하셔도 좋을거 같네요.

생성자를 빈 생성자를 만든 뒤, 해당 생성자에서 super(...) 로 값을 직접 채워넣는 방식으로 하면 사용하는측에선 더 깔끔하게 SCV에 대한 기본값을 몰라도 SCV를 만들 수 있습니다.